### PR TITLE
Setup tests

### PR DESCRIPTION
**Pull request by BorisOsipov**

### CODE REVIEW

File: src/test/java/integration/ITest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+
+import com.codeborne.selenide.Configuration;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+
+public class ITest {
+    @BeforeAll
+    static void beforeAll() {
+        Configuration.baseUrl = "https://borisosipov.github.io/test-page";
```
Comment (asolntsev) [inline src/test/java/integration/ITest.java:12]: @BorisOsipov Maybe move this testing page to https://selenide-selenoid.github.io?

File: src/test/java/integration/FileDownloadTest.java [hunk 2]
```diff
@@ -2,42 +2,60 @@
...
-  void setUp() {
-    closeWebDriver();
-
-    DesiredCapabilities capabilities = new DesiredCapabilities();
-    capabilities.setBrowserName("chrome");
-    Configuration.browserCapabilities = capabilities;
-    Configuration.fileDownload = FileDownloadMode.FOLDER;
+public class FileDownloadTest extends ITest {
+  @AfterEach
+  void tearDown() {
```
Comment (asolntsev) [inline src/test/java/integration/FileDownloadTest.java:26]: Maybe move this method to `ITest`?
And also add `@BeforeEach` annotation to it?

**Review by asolntsev — APPROVED** [review]

File: src/test/java/integration/ITest.java (hunk 1 above)
Comment (BorisOsipov) [reply src/test/java/integration/ITest.java:14]: @asolntsev Sure. Please enable github pages for branch `gh-pages` on this repo settings

Comment (asolntsev) [reply src/test/java/integration/ITest.java:14]: @BorisOsipov I have enabled it: https://selenide.org/selenide-selenoid/

P.S. Probably it's better to have some landing page (or at least some welcome text) on the page https://selenide.org/selenide-selenoid/, and put testing pages  to some subfolder etc.

Comment (asolntsev) [reply src/test/java/integration/ITest.java]: I updated the tests to run against https://selenide.org/test-page/

**Pull request merged**